recipes/geokit/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "geokit" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/geokit/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6d48d9adb1232e7f679d5e151ae004619ecb8270860e4d5a48ab764e80a0e6d0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-dependencies  .  -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Reply:
Why should the --no-dependecies flag be removed? I thought pip dependencies cause issues when combined conda dependencies  https://www.anaconda.com/blog/using-pip-in-a-conda-environment

Reply:
PS: that blog is about using pip in a conda environment, not about building recipes with conda-build. Please check our docs: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

Also, try to use grayskull to generate the recipe, which will do the right thing there, or follow our recipe example in the example folder.

Reply:
Thank you verry much for elaborating. I think the documentation is a bit missleading because the -vv comand only changes the output of the installation proceedure and does not directly exlclude the dependency installation from the command.

Also thank you for suggesting grayskull. I wasnt aware it can be used for libaries which are not published to pypi.